pkgs/development/tools/database/sqlfluff/default.nix

```diff
@@ -0,0 +1,65 @@
+{ lib
+, fetchFromGitHub
+, python3
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "sqlfluff";
+  version = "0.6.0a2";
+  disabled = python3.pythonOlder "3.6";
+
+  src = fetchFromGitHub {
+    owner = pname;
+    repo = pname;
+    rev = version;
+    sha256 = "13hzr0jialzi2nlvqwvff3w0h6jikqvcg0p2p4irwlisg4db8w7w";
+  };
+
+  propagatedBuildInputs = with python3.pkgs; [
+    appdirs
+    cached-property
+    click
+    colorama
+    configparser
+    diff_cover
+    jinja2
+    oyaml
+    pathspec
+    pytest
+    tblib
+    toml
+  ] ++ lib.optionals (pythonOlder "3.7") [
+    dataclasses
+  ] ++ lib.optionals (pythonOlder "3.9") [
```

Comment:
I think we only need this when it is under pythonPackages.

Reply:
Removing it would require to patch the source as `typing-extensions` is picked-up during the build process.